src/markdown/2021-01-30_function_composition_syntax.md

```diff
@@ -8,6 +8,10 @@
 
 _The execution order of code should match the left-to-right order it is read in English._
 
+_The execution order of code should match the left-to-right order as it is read in English._
+
+_The execution order of code should match the way it is read in English: from right to left._
```

Comment:
I like your second suggestion...after changing "right to left" to "left to right" ;)  I pushed a commit with that change.

Reply:
ah, derp x|